Lib/test/test_dbm.py

```diff
@@ -172,6 +176,10 @@ def test_whichdb_ndbm(self):
         with open(db_file, 'w'):
             self.addCleanup(os_helper.unlink, db_file)
         self.assertIsNone(self.dbm.whichdb(db_file[:-3]))
+        path = os_helper.FakePath(db_file)
+        with open(path, 'w'):
+            self.addCleanup(os_helper.unlink, path)
+        self.assertIsNone(self.dbm.whichdb(db_file[:-3]))
```

Comment:
It is already tested at line 178. You need to test for `FakePath(db_file[:-3])` and `fsencode(db_file[:-3])`.

Reply:
OK, I misunderstood that test.  Should be OK now.